filebeat/module/zookeeper/module.yml

```diff
@@ -0,0 +1,5 @@
+dashboards:
+- id: Filebeat-zookeeper-audit-Dashboard
+  file: Filebeat-zookeeper-audit.json
+- id: Filebeat-zookeeper-log-Dashboard
+  file: Filebeat-zookeeper-log.json
```

Comment:
If no dashboards are included please remove this.

Reply:
done